src/arduino-rfm/lorawan-arduino-rfm.cpp

```diff
@@ -242,58 +229,80 @@ void LoRaWANClass::setDevAddr(const char *devAddr_in)
 
     //Reset frame counter
     Frame_Counter_Tx = 0x0000;
-
-    //Reset RFM command status
-    RFM_Command_Status = NO_RFM_COMMAND;
 }
 
 void LoRaWANClass::setDeviceClass(devclass_t dev_class)
 {
     LoRa_Settings.Mote_Class = (dev_class == CLASS_A)? CLASS_A : CLASS_C;
 
-    if (LoRa_Settings.Mote_Class == CLASS_A) {
-        RFM_Switch_Mode(RFM_MODE_STANDBY);
-    } else {
-        RFM_Continuous_Receive(&LoRa_Settings);
-    }
+    if (LoRa_Settings.Mote_Class == 0x00) {
```

Comment:
I think we should choose define CLASS_A AND CLASS_C or the value hexadecimal and comment, here is hexadecimal